clothes-detection/src/App.js

```diff
@@ -1,12 +1,121 @@
-import React from 'react';
+import React, {Component} from 'react';
 import './App.css';
+import MagicDropzone from 'react-magic-dropzone';
 
-function App() {
-  return (
-    <div className="App">
-      <h1>Hi, I am react app.</h1>
-    </div>
-  );
+function getImageAttributes(imageUri) {
+  const yurieApiKey = 'AIzaSyCc2ZRO2cPmW50mHI8SDG1okNRzDo33gAU';
```

Comment:
[**A comment** (PR comments with a purpose)](https://gist.github.com/echolabstech/a905ad02efba1ef6663c96ee4f251480):
use your key and set restrictions for now, i'm disabling mine. [see class notes](https://gist.github.com/echolabstech/f5a7ce1d1d8a62c0cd505e94b9bac31c).